package.json

```diff
@@ -23,6 +23,8 @@
   },
   "homepage": "https://github.com/ballercat/wasm-loader#readme",
   "dependencies": {
+    "loader-utils": "^1.1.0",
+    "wasm-dce": "0.0.7"
```

Comment:
Could this be a peer dependency? I think that might be a bit better for the end user. That is, unless the API of `wasm-dce` is going to change much between versions, but looking at the code now it doesn't seem like it will.

Reply:
Yes, the API is really simple and I don't see why we would have to change it.

Reply:
Ok done, I hope I did it correctly tho!